dashboard/test/models/peer_review_test.rb

```diff
@@ -393,6 +395,29 @@ class PeerReviewTest < ActiveSupport::TestCase
     assert reviews.last.audit_trail.lines.last.include? "NO CONSENSUS after review by user id #{users.last.id}"
   end
 
+  test 'escalating a peer review creates a new review for an instructor to do' do
+    track_progress @level_source.id
+    reviews = PeerReview.last(2)
+    reviewers = create_list :teacher, 2
+
+    assert_creates(PeerReview) do
+      reviews.first.update(status: 'escalated', reviewer: reviewers.first)
+    end
+    assert_does_not_create(PeerReview) do
+      reviews.last.update(status: 'escalated', reviewer: reviewers.last)
+    end
+
+    new_review = PeerReview.last
+    assert_nil new_review.reviewer
+    assert_equal @user, new_review.submitter
+    assert_nil new_review.data
```

Comment:
Should we make [these](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/models/peer_review_test.rb#L381-L389) assertions more consistent?

Reply:
I think I have a way with the attributes method to do this